Updated test-driven blocks (#218)
validmind/documentation: site/guide/work-with-content-blocks.qmd

6. Click {{< fa square-plus >}} and then select one of the available options: 
    
    - **Simple text block**: Adds a new section with a blank content block. After the new content block has been added, click {{< fa pencil >}} to edit the contents of the section like any other.
    - **Test-driven block**: Adds a new section with test results. 

7. For test-driven blocks, a menu containing available tests will appear:

      ![](test-driven-block-menu.png){width=85%}

>    - Select tests using the checkboxes on the left.

Reviewer: ```suggestion
   - Select the tests to insert into the model documentation.
```

A good rule of thumb is to AVOID describing how the UI works but focus on the user task (here selecting tests to insert). Typically, if you have to describe the UI, there's a design issue.

   - Search by name using **{{<fa magnifying-glass >}} Search** on the top-left.

>    - Click on a test to preview it. By default, selected tests are previewed.

Reviewer: This text does something different than the other bullets — maybe make this a follow-on paragraph at the same indentation level as the bulleted list that says something like "To preview what is included in a test, select it. By default ..."

(AFAIK, selecting means clicking, so we should consistently use the same simplified action words.)

> 8. Click **Insert test result to document** when you are ready.

Reviewer: ```suggestion
8. Click **Insert _n_ test result to document** when you are ready.
```

It can be hard to match buttons that change based on what you select but this gets you a bit closer.
